client/packages/system/src/Vaccination/api/useVaccination.ts

```diff
@@ -218,17 +220,15 @@ const useUpdate = (vaccinationId: string | undefined) => {
         clinicianId: setNullableInput('id', input.clinician),
```

Comment:
Should Clinician id also be nulled if isOtherFacility?

Reply:
Oo great catch, thank you!